scalameta/semantic/src/main/scala/scala/meta/semantic/Errors.scala

```diff
@@ -0,0 +1,37 @@
+package scala.meta
+package semantic
+
+import org.scalameta.adt._
+import org.scalameta.data._
+import scala.meta.inputs._
+import scala.meta.internal.inputs._
+
+@root trait Completed[+T] {
```

Comment:
Internally, scalac calls the process of obtaining semantic information `completion`. Therefore, I just took this name and rolled with it. 

Judging by your reaction, this name is obscure, which is not good. Any ideas about replacing it?